### PR TITLE
designed the login component from Okta and matched the color scheme f…

### PR DESCRIPTION
**Pull request by JesusNavarrete215**
…rom coderHeroes

https://www.loom.com/share/c80047b0f60e4ff79540949baa692f28

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

co-author @alextcedwards

Fixes # (issue)

Please delete options that are not relevant.

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
\

## Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [ x] My changes generate no new warnings
- [x ] I have checked my code and corrected any misspellings
- [ x] No duplicate code left within changed files
- [x ] Size of pull request kept to a minimum
- [x ] Pull request description clearly describes changes made & motivations for said changes @

### CODE REVIEW

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -21,7 +25,7 @@ const LoginContainer = () => {
       // add your custom logo to your signing/register widget here.
       i18n: {
         en: {
-          'primaryauth.title': 'Welcome to Labs Basic SPA Please sign in',
+          'primaryauth.title': 'WELCOME BACK',
```
Comment (pjaepole) [inline src/components/pages/Login/LoginContainer.js:28]: For people that are interested to know how to style okta widget checkout 
https://developer.okta.com/docs/guides/custom-widget/main/#modify-strings

**Review by pjaepole — APPROVED** [review]
For people that are interested to know how to style okta widget checkout 
https://developer.okta.com/docs/guides/custom-widget/main/#modify-strings

It looks just like the mock-up awesome work

**Review by Ryan-E-Mark — APPROVED** [review]
Looks great! Awesome job replicating the mock-up. 

There are 30,000 files included in the package-lock.json, Nathan informed me if you copy the package-lock.json from main and paste into your branch and push up it should negate that.

Also it appears the social media icons aren't displaying properly on the navbar, not sure why that is but maybe something to do with ant design, might be worth looking into.

**Review by ElchinNasirov — APPROVED** [review]
Great job!

There some unused/commented code in src/styles/header.less.
Overall looks perfect, great design and very informative video. Well done.

File: src/styles/header.less [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    border-radius: 999px;
+width: 3em;
+height: 3em;
+}
+
+// .contact{
+//     background: white;
+//     border-radius: 3px;
+//     width: 8rem;
+//     height: 3rem;
```
Comment (paulstgermain) [inline src/styles/header.less:62]: I'd love to see this commented-out code removed in a future PR! 😉

**Pull request merged**